_includes/embed/audio.html

```diff
@@ -0,0 +1,25 @@
+{% assign supported_audio_extensions = 'mp3|ogg|wav|aac|webm|flac' %}
```

Comment:
Do we really need to maintain `supported_audio_extensions`?
Can't we just pass any extension to `convert-media-type.html`, so it may return ready to use MIME type? Or is it done to prevent passing wrong extensions and misprints to `additional_types`?

Reply:
Yes, It is. The `supported_(audio|video)_extnames` is needed in order to correctly specify file types and their media type to add the source tag. This way I am trying to avoid adding sources with the wrong data type for the audio or video tag.

Reply:
I decided to delete supported_(audio|video)_extnames.